exercises/practice/triangle/.docs/instructions.md

```diff
@@ -0,0 +1,23 @@
+# Instructions
+
+Determine if a triangle is equilateral, isosceles, or scalene.
+
+An _equilateral_ triangle has all three sides the same length.
+
+An _isosceles_ triangle has at least two sides the same length. (It is sometimes
+specified as having exactly two sides the same length, but for the purposes of
+this exercise we'll say at least two.)
+
+A _scalene_ triangle has all sides of different lengths.
+
+## Note
```

Comment:
We should add a note that this is going to test _both_ for integers and floating point numbers. That's an important detail for what I think this exercise is trying to teach (based on your example)

Reply:
Agreed. Added this note:

> Also, the triangle may be defined with integers or with real values which the
> program should also take into account.